e2e/cephfs_helper.go

```diff
@@ -53,20 +53,30 @@ func createCephfsStorageClass(c kubernetes.Interface, f *framework.Framework, en
 	if enablePool {
 		sc.Parameters["pool"] = "myfs-data0"
 	}
-	fsID, stdErr, err := execCommandInToolBoxPod(f, "ceph fsid", rookNamespace)
-	if err != nil {
-		return err
-	}
-	if stdErr != "" {
-		return fmt.Errorf("error getting fsid %v", stdErr)
+
+	// overload any parameters that were passed
+	if params == nil {
+		// create an empty params, so that params["clusterID"] below
+		// does not panic
+		params = map[string]string{}
+	} else {
+		for param, value := range params {
+			sc.Parameters[param] = value
+		}
```

Comment:
Else can be removed as we are intializing to empty map the for loop wont get executed.
```suggestion
if params == nil {
// create an empty params, so that params["clusterID"] below
// does not panic
   params = map[string]string{}
}
for param, value := range params {
     sc.Parameters[param] = value
}

Reply:
Sure, if that makes it better to read. Updated in the next version of this PR.